scripts/packaging/Makefile

```diff
@@ -106,7 +106,7 @@ install: all
 
 $(WEBOTS_DISTRO_EXE): webots_distro.c
 	+@echo "# compiling webots_distro.c"
-	+@$(CC) $(CFLAGS) -Wall -o $@ webots_distro.c $(LIBRARIES)
+	+@$(CC) $(CFLAGS) -Wno-deprecated-declarations -Wall -o $@ webots_distro.c $(LIBRARIES)
```

Comment:
I assume that deprecated declarations silenced here are specific to Ubuntu 22.04. If so, I would put this inside the Ubuntu 20.04 `CFLAGS`.